packages/edit-site/src/components/style-book/button-examples.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { __experimentalGrid as Grid } from '@wordpress/components';
+import { View } from '@wordpress/primitives';
+import {
+	BlockList,
+	privateApis as blockEditorPrivateApis,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { ExperimentalBlockEditorProvider, useGlobalStyle } = unlock(
+	blockEditorPrivateApis
+);
+
+const BUTTON_STATES = [
+	{
+		key: 'default',
+		title: __( 'Button' ),
```

Comment:
Since there's block title of "Button" do you think these labels could just describe the state? E.g., 'Default', 'Active', 'Hover' etc

<img width="711" alt="Screenshot 2024-12-06 at 1 38 00 pm" src="https://github.com/user-attachments/assets/df5eca95-afef-4059-bb9f-d0d6c6c221db">